app/controllers/collections_controller.rb

```diff
@@ -153,4 +156,17 @@ def set_collection_error_data
     @collection_format = ''
     @error = 'This collection is not available. It is either being published right now, does not exist, or you have insufficient permissions to view this collection.'
   end
+
+  def proposal_mode_enabled?
+    # actions available in both dMMT and MMT
```

Comment:
this may just be a preference thing, but it seems less confusing if the comment is nested under the `if` (and similar for the other comments here)